incubator/hnc/pkg/kubectl/root.go

```diff
@@ -123,16 +125,28 @@ func (cl *realClient) updateHierarchy(hier *api.HierarchyConfiguration, reason s
 	nnm := hier.Namespace
 	var err error
 	if hier.CreationTimestamp.IsZero() {
-		err = hncClient.Post().Resource(api.Resource).Namespace(nnm).Name(api.Singleton).Body(hier).Do().Error()
+		err = hncClient.Post().Resource(api.HierarchyConfigurations).Namespace(nnm).Name(api.Singleton).Body(hier).Do().Error()
 	} else {
-		err = hncClient.Put().Resource(api.Resource).Namespace(nnm).Name(api.Singleton).Body(hier).Do().Error()
+		err = hncClient.Put().Resource(api.HierarchyConfigurations).Namespace(nnm).Name(api.Singleton).Body(hier).Do().Error()
 	}
 	if err != nil {
 		fmt.Printf("\nCould not %s.\nReason: %s\n", reason, err)
 		os.Exit(1)
 	}
 }
 
+func (cl *realClient) createHierarchicalNamespace(nnm string, hnnm string) {
+	hns := &api.HierarchicalNamespace{}
+	hns.Name = hnnm
+	hns.Namespace = nnm
+	err := hncClient.Post().Resource(api.HierarchicalNamespaces).Namespace(nnm).Name(hnnm).Body(hns).Do().Error()
+	if err != nil {
+		fmt.Printf("\nCould not create hierarchicalnamespace instance.\nReason: %s\n", err)
```

Comment:
nit: Do we need space between "hierarchical" and "namespace"? Or change the message below to remove the space for consistency.